core/refinement/import_methods_spec.rb

```diff
@@ -0,0 +1,34 @@
+require_relative '../../spec_helper'
+
+describe "Refinement#import_methods" do
+  ruby_version_is "3.1" do
+    context "when methods are defined in Ruby code" do
+      it "imports methods" do
+        str_utils = Module.new do
+          def indent(level)
+            " " * level + self
+          end
+        end
+
+        Module.new do
+          refine String do
+            import_methods str_utils
+            "foo".indent(3).should == "   foo"
+          end
+        end
+      end
+    end
+
+    context "when methods are not defined in Ruby code" do
+      it "raises ArgumentError" do
+        Module.new do
+          refine String do
+            -> {
+              import_methods Enumerable
```

Comment:
`Enumerable` is actually likely to be implemented in Ruby, it is in TruffleRuby for instance.
`Kernel` is unlikely to be implemented entirely in Ruby (e.g., object_id, send), could you use `Kernel`?

Reply:
Thanks you for your comment. I fixed the spec.